stats/src/test/java/io/airlift/stats/TestTDigest.java

```diff
@@ -257,6 +260,25 @@ public void testMerge()
         assertEquals(merged.valueAt(1), 8.0);
     }
 
+    @Test
+    public void testUnmergeable()
+            throws IOException
```

Comment:
We normally just do `throws Exception` for test methods

Reply:
I generally try to use the most specific exception the test can throw (plus, that's what intellij adds when it's missing the required declaration)